### PR TITLE
Update User Guide - Use Cache Options from WMS service

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Add Use Cache Options on [Catalog](https://mapstore--10362.org.readthedocs.build/en/10362/user-guide/catalog/#advanced-settings_2) section 

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: docs/user-guide/catalog.md [hunk 1]
```diff
@@ -233,6 +233,8 @@ Enabling that option, all layers added to the map from this catalog source will
 !!! note
     If the **No Vendor** is set, then [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) will not use any vendor option supported only by GeoServer in the OGC requests where this source is involved.
 
+* *Use remote custom tile grids*: allows use of the layer cached tiles. If checked, the *Tiled=true* URL parameter will be added to the WMS request to [use tiles cached with GeoWebCache](https://docs.geoserver.org/latest/en/user/geowebcache/using.html#direct-integration-with-geoserver-wms), these will be automatically applied to the [Layer Settings](layer-settings.md#display) when a layer is added to the map from this source.
```
Comment (tdipisa) [inline docs/user-guide/catalog.md:236]:
```suggestion
* *Use remote custom tile grids*: if checked, allows to set by default the _custom tile grid caching strategy_ when a layer is added to the map from this source as described in [Layer Settings](layer-settings.md#display).
```

**Review by tdipisa — CHANGES_REQUESTED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo, changed applied. Please backport to 2024.01.xx. Thank you.

**Review by tdipisa — APPROVED** [review]

**Review by ElenaGallo — COMMENTED** [review]
*Use remote custom tile grids*: if checked, allows to set by default the *custom tile grid caching strategy* when a layer is added to the map from this source as described in [Layer Settings](layer-settings.md#display).

**Review by tdipisa — APPROVED** [review]

**Comment by tdipisa** [comment]
> @ElenaGallo, changed applied. Please backport to 2024.01.xx. Thank you.

Merged @ElenaGallo please proceed.

**Pull request merged**